SPECS/ig/ig.signatures.json

```diff
@@ -1,6 +1,6 @@
 {
  "Signatures": {
-  "ig-0.30.0-govendor-v1.tar.gz": "b2db05dc0389ef7a4456f3a04edc96d2c89c4c599e89db39ce3b0d14314d60e9",
-  "ig-0.30.0.tar.gz": "75422f8f8a596019724f65acaa5165c142d9c580334114b3f7a7f86c742ee198"
+  "ig-0.32.0-govendor-v1.tar.gz": "daeb58510a1796dd76a78ac61c421630a61f5589f0ee3f35949500d731edb7bd",
```

Comment:
Uploaded the src & vendor to blobstore:
- https://dev.azure.com/mariner-org/mariner/_build/results?buildId=633718&view=results
- https://dev.azure.com/mariner-org/mariner/_build/results?buildId=633707&view=results